Document/0x05b-Basic-Security_Testing.md

```diff
@@ -105,12 +117,21 @@ The CA certificates trusted by the app can be a system trusted CA as well as a u
 
 To implement this new setting you must follow the steps below:
 
-- Decompile the app using decompilation tools.[Manual static Analysis](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05b-Basic-Security_Testing.md#manual-static-analysis) provides details about decompiling
-- Make the application trust user's certificates by creating a network security configuration that includes `<certificates src="user" />` as explained above
-- Repackage the app. The [Android developer documentation](https://developer.android.com/studio/publish/app-signing#signing-manually) explains how it's done.
+- Decompile the app using a decompilation tool like apktool:
+```bash
+$ apktool d <filename>.apk
+```
+- Make the application trust user certificates by creating a network security configuration that includes `<certificates src="user" />` as explained above
```

Comment:
It's hard to see from this review page, but does the section above explain both creating the xml file in thee correct resources folder, and how it should be linked from within the manifest?

Reply:
Now it does :-) I linked to this page that explains it very well https://codelabs.developers.google.com/codelabs/android-network-security-config/#3